qiskit/algorithms/optimizers/de.py

```diff
@@ -0,0 +1,134 @@
+                'recombination', 'init', 'tol', 'atol',
+                'polish', 'disp', 'workers']
+
+    def __init__(self,
+                 strategy: str = 'currenttobest1bin',
+                 maxiter: int = 10,
+                 popsize: int = 10,
+                 mutation: Union[float, tuple] = (0.5, 1),
+                 recombination: float = 0.7,
+                 init: str = 'latinhypercube',
+                 tol: float = 0.01,
+                 atol: float = 0,
+                 polish: bool = True,
+                 disp: bool = False,
+                 workers: Union[int, Callable] = 1) -> None:
+        """
+        Args:
+            strategy: Differential evolution strategy to employ for generating new trial candidates
+                      in subsequent generations/iterations
+            maxiter: Maximum number of generations to evolve population
+            popsize: Number of individuals/candidates in each generation
+            mutation: Mutation constant (differential weight) specified as a float in the range of
+                      [0,2] or as a tuple(min, max). If specified as a tuple, the mutation constant
+                      will be 'dithered' or randomly changed within the range specified in each
+                      generation.
+            recombination: Recombination constant (crossover probability) specified as a float
+                           in the range [0,1]
+            init: String/array to specify whether initial population is generately via ``random``,
+                  ``latinhypercube`` sampling or via a specified array of individuals. If an array
+                  is specified, its shape must match (popsize, params) where popsize and params
+                  refer to population size and number of parameters in the objective function
+            tol: Relative tolerance for convergence
+            atol: Absolute tolerance for convergence
+            polish: Boolean to specify whether or not best population in last generation is
+                    optimized using scipy.optimize.minimize with the L-BFGS-B method
+            disp: Boolean to specify if the objective function is printed at each iteration
+            workers: Integer or map-like callable (multiprocessing.Pool.map) that specifies
+                     how many individuals in the population to be evaluated in parallel
+        """
+
+        super().__init__()
+
+        self._strategy = strategy
+        self._maxiter = maxiter
+        self._popsize = popsize
+        self._mutation = mutation
+        self._recombination = recombination
+        self._tol = tol
+        self._atol = atol
+        self._init = init
+        self._polish = polish
+        self._disp = disp
+        self._workers = workers
+
+    def get_support_level(self) -> Dict[str, OptimizerSupportLevel]:
+        """ Support level dictionary
+
+        Returns:
+            Dict[str, int]: gradient, bounds and initial point

```

Comment:
The type hint is not required in the docstring since it's already provided above.

Reply:
Great, fixed!